packages/components/psammead-livel-label/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-live-label",
-  "version": "0.1.0-alpha.2",
+  "version": "0.1.0",
```

Comment:
I think we should change this to 1.0.0

Reply:
I think this is correct, since we've decided to start to use `0.1.0` as the initial version of the packages. You can see this for example in the `SectionLabel`.

Reply:
ah okay 👍

Reply:
On the other hand, there are a couple of components that were bumped to `1.0.0` from the alpha version, as you suggested, but I think that is wrong.

Reply:
Update major version instead as per https://github.com/bbc/psammead/pull/3293